app/serializers/v1/billing_entity_serializer.rb

```diff
@@ -26,14 +26,6 @@ def serialize
         document_number_prefix: model.document_number_prefix,
         tax_identification_number: model.tax_identification_number,
         finalize_zero_amount_invoice: model.finalize_zero_amount_invoice,
-        billing_configuration:
```

Comment:
It's a breaking change in the API contract. Is it expected?

(I agree that it does look better :slightly_smiling_face: )

Reply:
it's not a breaking change because it's a completely new model 😁 
we're leaving organization as is, but for the new billing entities we'll be serializing billing_configuration the new way

Reply:
Oh indeed, sorry for the noise :D